### ISSUE TITLE
coding-standard.xhtml describes old copyright header format

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5112]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ivank reported|
|-|-|
|Trac ID|trac#5112|
|Type|defect|
|Created|2011-05-18 19:22:42Z|
|Branch|https://github.com/twisted/twisted/tree/copyright-policy-5112|

http://twistedmatrix.com/trac/browser/trunk/doc/core/development/policy/coding-standard.xhtml?format=raw says:

```
# Copyright (c) 2009 Twisted Matrix Laboratories.
# See LICENSE for details.


    When you update existing files, make sure the year in the copyright
    header is up to date as well. You should add a new copyright header when
    it's completely missing in the file that is being edited.
```

We don't have years in the copyright headers any more.

<details><summary>Searchable metadata</summary>

```
trac-id__5112 5112
type__defect defect
reporter__ivank ivank
priority__normal normal
milestone__ 
branch__branches_copyright_policy_5112 branches/copyright-policy-5112
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1305746562000000 1305746562000000
changetime__1306083655000000 1306083655000000
version__None None
owner__exarkun exarkun
cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [31925]) Branching to 'copyright-policy-5112'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Thanks for pointing that out.  Fix is in the branch.

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|ivank commented|
|-|-|

1. Please fix "including the year the file was added" as well.

Otherwise, looks good.

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|ivank set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [31930]) Oops, remove the other year-related recommendation

refs [#5112](https://github.com/twisted/twisted/issues/5112)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [31931]) Merge copyright-policy-5112

Author: exarkun
Reviewer: ivank
Fixes: [#5112](https://github.com/twisted/twisted/issues/5112)

Remove the recommendation to include the year in copyright headers, matching
what we currently actually want to do with respect to these.  See [#4857](https://github.com/twisted/twisted/issues/4857).
